lib/src/git_backend.rs

```diff
@@ -138,8 +138,20 @@ impl GitBackend {
         let extra_path = store_path.join("extra");
         fs::create_dir(&extra_path).context(&extra_path)?;
         let target_path = store_path.join("git_target");
-        fs::write(&target_path, git_repo_path.to_str().unwrap().as_bytes())
-            .context(&target_path)?;
+        if cfg!(windows) && git_repo_path.is_relative() {
+            // WSL doesn't like backslashes in paths, so we manually format the path with
+            // forward slsashes (non-WSL is fine with forward slashes). If the path is
+            // absolute, there's much we can do - it simply won't work on both inside and
+            // outside WSL.
```

Comment:
Typo on `slsashes`, but I'd probably just reword this to be a bit more clear about the exact condition under which an error occurs:

```
// When a repository is created in Windows, format the path with *forward slashes*
// and not backwards slashes. This makes it possible to use the same repository under
// Windows Subsystem for Linux.
```

Reply:
Done (with a few additional typo fixes)